.circleci/config.yml

```diff
@@ -103,3 +137,44 @@ jobs:
       - run:
           name: Build the documentation.
           command: nox -s docs
+  publish_package:
+    docker:
+      - image: python:3.7-slim
+    steps:
+      - checkout
+      - run:
+          name: Decrypt the PyPI key.
+          command: |
+            openssl aes-256-cbc -d \
+              -in .circleci/.pypirc.enc \
+              -out ~/.pypirc \
+              -k "${PYPIRC_ENCRYPTION_KEY}"
+      - run:
+          name: Publish to PyPI.
+          command: python setup.py sdist upload
+  publish_image:
+    docker:
+      - image: google/cloud-sdk
+    steps:
+      - checkout
+      - run:
+          name: Build Docker image.
+          command: docker build . -t gcr.io/gapic-images/gapic-generator-python:latest
```

Comment:
To run docker inside docker, you need to do something special. Example here: https://github.com/googleapis/artman/blob/a429a2c05238fb65470e9e490ee83cec64bc8a56/.circleci/config.yml#L101

Reply:
Done.